phlex/core/concepts.hpp

```diff
@@ -55,13 +54,11 @@ namespace phlex::experimental {
 
   template <typename T>
   concept is_output_like = std::is_member_function_pointer_v<T> &&
-                           expects_input_parameters<T, product_store const&> &&
-                           returns<T, void>;
+                           expects_input_parameters<T, product_store const&> && returns<T, void>;
 
   template <typename T>
   concept is_fold_like =
-    at_least_two_input_parameters<T> &&
-    first_input_parameter_is_non_const_lvalue_reference<T> &&
+    at_least_two_input_parameters<T> && first_input_parameter_is_non_const_lvalue_reference<T> &&
```

Comment:
Is there a way to require that concepts are formatted with only one logical clause per line, and is this something we should do?